shared/utils/parseDocumentSlug.ts

```diff
@@ -11,5 +11,7 @@ export default function parseDocumentSlug(url: string) {
     }
   }
 
-  return parsed.replace(/^\/doc\//, "");
+  return parsed.lastIndexOf("/doc/") === 0
```

Comment:
gonna need context/tests for this change 😄

Reply:
`- [ ] Setup the [Slack integration](/settings/integrations/slack)`

In the Getting started doc, hovering over this led to API request with `/settings/integrations/slack` as Id.